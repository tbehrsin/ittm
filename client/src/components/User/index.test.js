import React from 'react';
import { render } from '@testing-library/react';
import { User } from './index';
import mockUsers from '../../../../mock-object.json';

mockUsers.data.items.forEach(item => {
  item.id = item.id.toString();
  item.role.id = item.role.id.toString();
});

describe('User', () => {
  test('renders without crashing', () => {
    const { asFragment } = render(
      <table>
        <tbody>
          <User user={mockUsers.data.items[0]} />
        </tbody>
      </table>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
