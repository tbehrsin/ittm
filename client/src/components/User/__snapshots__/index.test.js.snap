// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User renders without crashing 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr
        class="container"
      >
        <td>
          123
        </td>
        <td>
          support_admin@example.com
        </td>
        <td>
          UK
        </td>
        <td>
          false
        </td>
        <td>
          ACTIVE
        </td>
        <td>
          Monday 6th April 2020
        </td>
        <td>
          1
        </td>
        <td>
          test role
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
